core/src/test/java/org/wildfly/extras/sunstone/tests/AbstractCloudProviderTest.java

```diff
@@ -155,19 +157,28 @@ public final void node() throws Exception {
                 // OK
             }
 
-            try (Node node = cloudProvider.createNode("mynode")) {
-                assertEquals(1, cloudProvider.getNodes().size());
-                assertNotNull(cloudProvider.getNode("mynode"));
+            try (CreatedNodes createdNodes = cloudProvider.createNodes("mynode", "othernode")) {
```

Comment:
I don't think that removing "othernode" would lower test coverage. A collection of one is not that different from a collection of two, if they are used the same way.

Anyway, if you're fine with the test's performance right now, with two nodes, I'm OK with that too :-)